shanoir-uploader/src/main/java/org/shanoir/uploader/nominativeData/CurrentNominativeDataController.java

```diff
@@ -44,20 +44,22 @@ public class CurrentNominativeDataController {
 
 	private CurrentUploadsWindowTable cuw;
 
+	@SuppressWarnings("deprecation")
 	public void configure(final File workFolderFilePath, final CurrentUploadsWindowTable cuw) {
 		this.currentNominativeDataModel = new CurrentNominativeDataModel();
 		currentNominativeDataModel.addObserver(cuw);
 		this.cuw = cuw;
 		processWorkFolder(workFolderFilePath);
 
-		cuw.table.addMouseListener(new MouseAdapter() {
-			public DefaultTableModel model = (DefaultTableModel) cuw.table.getModel();
+		CurrentUploadsWindowTable.table.addMouseListener(new MouseAdapter() {
```

Comment:
good! I would remove the cuw attribute, when you pass directly by the static attribute of CUWT?

Reply:
even remove the param from configure

Reply:
I modified the instantiation of cut as a true singleton loaded once and for all by ReadyState.load()